x/evm/keeper/params_test.go

```diff
@@ -101,6 +106,46 @@ func (suite *KeeperTestSuite) TestParams() {
 			},
 			true,
 		},
+		{
+			"success - EnabledPrecompiles param is set to empty slice and will be retrieved as nil",
+			func() interface{} {
+				params.EnabledPrecompiles = []string{}
+				suite.app.EvmKeeper.SetParams(suite.ctx, params)
```

Comment:
Seems like `TestParams` doesn't directly check the`SetParams` error return value, not sure if we want to refactor it to use it 

Additional tests for failure cases would be good to ensure the `SetParams` is actually calling the Validate() method properly

Reply:
Fixed: https://github.com/Kava-Labs/ethermint/pull/55/commits/c8f023e74aac2dc0fb88cdfa5865c7316661c1af